### PR TITLE
Bugfix in integration test for Build Plan

### LINKED ISSUE #174: I3S - Integration test for Build Plan failing
### Scenario/Intent
The integration test for the build plan was not possible to test the appliance error. After reimaging the appliance, the test was performed and errors were found due to object search that did not exist.

### Environment Details
 - **OneView SDK Version: current master repo
 - **OneView Appliance Version: 3.0
 - **OneView Client API Version: 300
 - **Ruby Version: 2.2.1+
 - **Platform: Any

### Steps to Reproduce
Run integration test for Build Plan.

### Expected Result
The delete test performed successfully.

### Actual Result
The delete test failing.

### PR DESCRIPTION
**Pull request by aalexmonteiro**
### Description
The integration test for the build plan was not possible to test the appliance error. After reimaging the appliance, the test was performed and errors were found due to object search that did not exist.

### Issues Resolved
#174 

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Comment by tmiotto** [comment]
@aalexmonteiro  don't you think this attribute missing in the integration test shouldn't be also highlighted on the `examples` ?

**Comment by aalexmonteiro** [comment]
@thiago, you're right. Good hint. :)

**Comment by aalexmonteiro** [comment]
@tmiotto , I added the example with custom attributes.

File: examples/image-streamer/api300/build_plan.rb [hunk 1]
```diff
@@ -12,16 +12,20 @@
...
+#   @plan_script2_name (plan script with build step and custom attributes)
 
 options = {
   name: 'Build_Plan_1',
   oeBuildPlanType: 'Deploy'
 }
 
-plan_script = OneviewSDK::ImageStreamer::API300::PlanScript.find_by(@client, name: @plan_script_name).first
+plan_script = OneviewSDK::ImageStreamer::API300::PlanScript.find_by(@client, name: @plan_script1_name).first
+plan_script2 = OneviewSDK::ImageStreamer::API300::PlanScript.find_by(@client, name: @plan_script2_name).first
```
Comment (jsmartt) [inline examples/image-streamer/api300/build_plan.rb]: Update the _client_i3s.rb.example file with these new names.

Comment (aalexmonteiro) [inline examples/image-streamer/api300/build_plan.rb]: @jsmartt , changes made.

**Pull request merged**